doc/api/stream.md

```diff
@@ -1510,6 +1510,39 @@ class MyWritable extends Writable {
 }
 ```
 
+#### Decoding buffers in a Writable Stream
+
+Decoding buffers is a common task, for instance, when using transformers whose
+input is a string. This is not a trivial process when using multi-byte
+characters encoding. Implement it within [Writable][] implies some performance
```

Comment:
I would add: "...multi-byte characters encoding, __such as UTF-8__."

I would remove the sentence "Implement it within Writable implies some performance regressions".

Then:

"The following example shows how to decode multi-byte strings using `StringDecoder` and `Writable`."